src/EditorFeatures/Test/SolutionCrawler/WorkCoordinatorTests.cs

```diff
@@ -1058,6 +1084,18 @@ protected override void Dispose(bool finalize)
                 Assert.False(_workspaceWaiter.HasPendingWork);
                 Assert.False(_solutionCrawlerWaiter.HasPendingWork);
             }
+
+            private static IExportProviderFactory CreateExportProviderFactory()
+            {
+                var assemblies = TestExportProvider
+                    .GetCSharpAndVisualBasicAssemblies()
+                    .Concat(new[] { typeof(EditorServicesUtil).Assembly });
+
+                return ExportProviderCache.GetOrCreateExportProviderFactory(
+                    ExportProviderCache.GetOrCreateAssemblyCatalog(
+                        assemblies, ExportProviderCache.CreateResolver()).WithPart(
+                        typeof(TestGlobalOperationService)));
```

Comment:
:memo: This code formatting suggests that `assemblies` and `typeof(TestGlobalOperationService)` are two parameters to the same method. This is not the case here; a better layout would be this:

```csharp
return ExportProviderCache
  .GetOrCreateExportProviderFactory(assemblies, ExportProviderCache.CreateResolver())
  .WithPart(typeof(TestGlobalOperationService));
```

Reply:
not sure 100% I understand. but sure, let me change it.

Reply:
GetOrCreateExportProviderFactory accepts ComposableCatalog and GetOrCreateAssemblyCatalog is the one that creates one. not sure how I do this.

return ExportProviderCache
  .GetOrCreateExportProviderFactory(assemblies, ExportProviderCache.CreateResolver())
  .WithPart(typeof(TestGlobalOperationService));

can you give me some more detail?

Reply:
I accidentally left out the call to `GetOrCreateAssemblyCatalog`. It should have been this:

```csharp
return ExportProviderCache
  .GetOrCreateExportProviderFactory(
    ExportProviderCache.GetOrCreateAssemblyCatalog(assemblies, ExportProviderCache.CreateResolver()))
  .WithPart(typeof(TestGlobalOperationService));
```

Reply:
so, you want to change format of the code?